### ISSUE TITLE
Not Working Except Image Labeling

### ISSUE THREAD

**Issue by CHIRANJIT1988** [husaynhakeem/Android-ML-Kit-Sample#2]
Hi, have cloned and installed on my device. The image labeling feature is working fine and rest of the feature face detection, barcode etc are not working. What may be the issue. Please help me on this. Thanks

**Comment by husaynhakeem** [comment]
Hi,
My guess is that the issue is with the photos you're using. Please share them with me so I can test on my side.

**Comment by CHIRANJIT1988** [comment]
Thanks. I have checked so many images non of them are working. Please find one of them.

![img-20180529-wa0005](https://user-images.githubusercontent.com/15885223/40701974-ab93f0f2-63fd-11e8-8d65-2cb5484104bd.jpg)

**Comment by husaynhakeem** [comment]
Hey @CHIRANJIT1988, sorry for getting back a bit late.
So I just tried a couple of times with your photo, it seems to be working fine (except for the text detection, it's not great, but I had noticed this issue previously with other photos, so the results are pretty consistent). Below are the results I got. I'm not sure why it isn't working right for you, maybe this has to do with the device you're using (?).
![screenshot_20180612-110152](https://user-images.githubusercontent.com/7649257/41308433-b5d22ae2-6e30-11e8-8200-63db1ca4eaad.jpg)
![screenshot_20180612-110201](https://user-images.githubusercontent.com/7649257/41308435-b5ee8070-6e30-11e8-80c5-98221b2677dd.jpg)
![screenshot_20180612-110231](https://user-images.githubusercontent.com/7649257/41308437-b60ba4ac-6e30-11e8-9000-1882cdc03940.jpg)
